bude_hezky/sender/email_sender.py

```diff
@@ -0,0 +1,30 @@
+import os
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail
+
+SAFE_STATUS_CODE_THRESHOLD = 299
+
+class EmailNotSentException(Exception):
+    pass
+
+def sendMail(fromEmail, toEmail, subject, content, send_grid_client=None):
```

Comment:
nitpick: camelCase

Reply:
Přijde mi, že tahle funkce dělá hodně věcí a nepůjde testovat. Převrať to myšlení. Dej side effecty (requesty, posílání mailu) do toho bordelu třeba v main a všechno hezké a čisté si vyhoď ven a pěkně otestuj. Teď je to pořád dost naopak, i když cením, že main aspoň vcucnul ty command line argumenty, teď to jde otevřít a hned vidíš co ten program bere a jak funguje a až potom se můžeš začít nořit do dalších souborů.